### PR TITLE
[QA-1837] Move jest-junit config

### PR DESCRIPTION
**Pull request by aweng98**
Jest's configuration are defined in the `package.json` file and `jest.config.js` file. Either file is fine but I feel it's wrong to be in  both files. 
Clean up `package.json`, move `jest-junit` config to `jest.config.js`.

https://broadworkbench.atlassian.net/browse/QA-1837

### CODE REVIEW

File: .circleci/config.yml [hunk 1]
```diff
@@ -155,7 +155,7 @@ commands:
           command: |
             mkdir -p ${SCREENSHOT_DIR}
             TESTS_TO_RUN=$(circleci tests glob "jest/**.integration-test.js" | circleci tests split --split-by=timings)
-            yarn test $TESTS_TO_RUN --maxWorkers=2 --reporters=default --reporters=jest-junit
```
Comment (aweng98) [inline .circleci/config.yml:158]: The `reporters` flag is redundant after the move.

File: integration-tests/jest.config.js [hunk 2]
```diff
@@ -1,4 +1,16 @@
 module.exports = {
+  verbose: false,
```
Comment (aweng98) [inline integration-tests/jest.config.js:2]: Jest bug workaround related to `includeConsoleOutput` flag.
https://github.com/jest-community/jest-junit/issues/45#issuecomment-514398747

**Review by kyuksel — APPROVED** [review]
LGTM. May want to post on #terra-ui for the second review.

**Review by petesantos — APPROVED** [review]

**Comment by aweng98** [comment]
Thank you so much, @kyuksel @petesantos

**Pull request merged**